pootle/apps/pootle_project/models.py

```diff
@@ -244,9 +245,6 @@ def accessible_by_user(cls, user):
             allow_usernames = list(set([username, 'default', 'nobody']))
             forbid_usernames = list(set([username, 'default']))
 
-        # FIXME: use `cls.objects.cached_dict().keys()`, but that needs
```

Comment:
This note is not obsolete: it's about using the values from the cache directly, i.e
```python
ALL_PROJECTS = cls.objects.cached_dict(user).keys()
```
as opposed to the current
```python
ALL_PROJECTS = cls.objects.values_list('code', flat=True)
```

Feel free to address that separately, however as said the comment at the moment is not obsolete.